examples/marine/asv/rex/RadarFactor.jl

```diff
@@ -61,30 +62,36 @@ function (rp2::AlignRadarPose2)(res::Vector{Float64},
 end
 
 struct PackedAlignRadarPose2 <: PackedInferenceType
-  im1::Vector{Vector{Float64}}
-  im2::Vector{Vector{Float64}}
+  im1::String
+  im2::String
   PreSampler::String
   p2p2::PackedPose2Pose2
 end
 
+function image2string(image::Array{Float64, 2})::String
+  io = IOBuffer()
+  save(Stream(format"PNG", io), image)
+  return base64encode(io.data)
+end
+
+function string2image(stringdata::String)::Array{Float64, 2}
+  io = IOBuffer(base64decode(stringdata))
```

Comment:
Just to help string together all the different image encoding methods -- here is a binary store via png (ImageMagick.jl):
https://juliarobotics.org/Caesar.jl/latest/concepts/interacting_fgs/#Adding-Data-Blobs 

I also created this in Caesar.jl
https://github.com/JuliaRobotics/Caesar.jl/blob/7ec2d116e901816bfddbb4018bb043df3a4184b7/src/images/imagedata.jl#L26